{
    "body": "I've been having problems building v1.3.0 due to checksum mismatch and I confirm that the checksum in the package oddly no longer matches that calculated by spack.\r\n\r\n### Steps to reproduce the issue\r\n\r\n```console\r\n$ spack checksum py-hatchet@1.3.0\r\n==> Found 1 version of py-hatchet:\r\n  \r\n  1.3.0  https://github.com/hatchet/hatchet/archive/v1.3.0.tar.gz\r\n\r\n==> Fetching https://github.com/hatchet/hatchet/archive/v1.3.0.tar.gz\r\n######################################################################## 100.0%\r\n\r\n    version('1.3.0', sha256='d77d071fc37863fdc9abc3fd9ea1088904cd98c6980a014a31e44595d2deac5e')\r\n```\r\n\r\nBUT the checksum in the package is:\r\n\r\n```\r\n    version('1.3.0', sha256='d38212aec6272558e25538ff23b54ec0f2b4dbf6ef4b0e77f7f05c37c0d17815')\r\n```",
    "user": "tldahlgren",
    "url": "https://api.github.com/repos/spack/spack/issues/23901",
    "updated_at": "2021-05-27 16:40:34",
    "created_at": "2021-05-25 21:55:06",
    "closed_at": "2021-05-27 16:40:34",
    "state": "closed",
    "title": "Installation issue: py-hatchet v1.3.0 checksum mismatch",
    "number": 23901,
    "milestone": null,
    "labels": [
        "build-error"
    ],
    "id": 901417274,
    "html_url": "https://github.com/spack/spack/issues/23901",
    "assignees": [],
    "comments": 1
}